src/mono/wasm/build/README.md

```diff
@@ -140,4 +140,17 @@ If encountering build performance issues, you can use the rollup `--perf` option
 
 ```node node_modules/typescript/lib/tsc.js --generateCpuProfile dotnet-tsc.cpuprofile -p tsconfig.json ```
 
-The .cpuprofile file generated by node can be opened in the Performance tab of Chrome or Edge's devtools.
\ No newline at end of file
+The .cpuprofile file generated by node can be opened in the Performance tab of Chrome or Edge's devtools.
+
+## Blazor
+
+You can test local build of runtime in blazor by overriding location where blazor resolves runtime pack from
+
+*BlazorWasm.csproj*
+```xml
+<Target Name="UpdateRuntimePack" AfterTargets="ResolveFrameworkReferences">
+  <ItemGroup>
+    <ResolvedRuntimePack PackageDirectory="{RUNTIME_REPO_ROOT}\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Debug" Condition="'%(ResolvedRuntimePack.FrameworkName)' == 'Microsoft.NETCore.App'" />
```

Comment:
for testing AOT do you also need to point at a different Cross AOT package?

Reply:
For an ItemGroup don't you usually need `Include=` or `Update=` ? what does it do if you don't one of those?

Reply:
Update works only if items are defined outside of a target. This updates all items matching the condition

Reply:
> for testing AOT do you also need to point at a different Cross AOT package?

Yes - I don't know if this is too early, but you could clear the `MonoAotCrossCompiler` item and provide a new one that points in tree.

Reply:
I have never done that, you have a snippet for it?